### PR TITLE
Add precedent info for service token path and value in agent container

### PR DESCRIPTION
**Pull request by michel-laterman**
Add a description of how the service token value and path env vars are used in agent container mode.

Follow up to: https://github.com/elastic/elastic-agent/pull/2576

### CODE REVIEW

**Comment by michel-laterman** [comment]
Do we have backport tags? This should be part of the 8.8 docs

**Review by ycombinator — APPROVED** [review]
LGTM.

File: docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc [hunk 1]
```diff
@@ -172,7 +172,8 @@ Overrides `FLEET_TOKEN_POLICY_NAME` when set.
 [id="env-{type}-fleet-server-service-token"]
 `FLEET_SERVER_SERVICE_TOKEN`
 
-| (string) Service token to use for communication with {es}.
+| (string) Service token to use for communication with {es} and {kib} if <<kibana-fleet-setup>> is enabled.
```
Comment (kilfoyle) [inline docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc:175]:
```suggestion
| (string) Service token to use for communication with {es} and {kib} if <<env-prepare-kibana-for-fleet,`KIBANA_FLEET_SETUP`>> is enabled.
```
This fixes the link to point to the "[Prepare Kibana for Fleet](https://www.elastic.co/guide/en/fleet/8.7/agent-environment-variables.html#env-prepare-kibana-for-fleet)" anchor, which should fix the failing Docs CI checks.

**Comment by michel-laterman** [comment]
@kilfoyle do we need to manually backport this for 8.8?

**Comment by kilfoyle** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.8|[](https://github.com/elastic/ingest-docs/pull/193)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by kilfoyle** [comment]
@michel-laterman We don't have backport labels set up in this repo, at least not yet. For now, we've been using the [backport tool](https://github.com/sqren/backport).

Install:
`npm install -g backport`
and then:
`backport --pr {PR_NUM} --branch {branch_name}`

You can just let any of us writers know whenever a docs PR needs backporting, but I'll also try to find out how to set up labels, to make this easier.

I opened https://github.com/elastic/ingest-docs/pull/193 to get this into the 8.8 docs. I'll merge that as soon as it passes CI checks.

**Pull request merged**